inferential/static/playground.js

```diff
@@ -59,3 +59,30 @@ async function displayCompletion() {
   button.disabled = false
   loadingSpinner.classList.add('spinner-hidden')
 }
+
+//Displays text from prompts based on which model is selected
+const promptLong = document.getElementById('promptLong')
```

Comment:
I believe we could put all these variables in an object or an arrray of sorts that way we don't have declare all of these one by one and you can have better manipulation of these variables

Reply:
From my research, playground.js cannot connect to config.yml directly, it pulls its data directly from playground.html in the same way that the model select is pulled directly from config.yml.

playground.js gets the data from models from config.yml, and to do that it connects the same way the select models drop down does. Creating an array would make more work not really anything.